### PR TITLE
containerd config improvement

### PR DESCRIPTION
**Pull request by RA489**
containerd config improvement.

fixes https://github.com/kubernetes/website/issues/22591

/language en

### CODE REVIEW

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -445,7 +445,11 @@ systemctl restart containerd
 
 ### systemd
 
-To use the `systemd` cgroup driver, set `plugins.cri.systemd_cgroup = true` in `/etc/containerd/config.toml`.
+To use the `systemd` cgroup driver, go to `/etc/containerd/config.toml` and set
+
+[plugins.cri]`
+
+`systemd_cgroup = true`
 When using kubeadm, manually configure the
```
Comment (bart0sh) [inline content/en/docs/setup/production-environment/container-runtimes.md:453]: Here is how it looks in preview:
Screen Shot 2020-08-20 at 10 36 58

It should be reformatted a bit to be more readable, I believe.

**Review by neolit123 — COMMENTED** [review]
Thanks. This clarification is a good idea. Leaving LGTM to Ed.

/sig cluster-lifecycle

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/production-environment/container-runtimes.md:453]: To be honest, @RA489, I prefer the original text.

Comment (celestehorgan) [reply content/en/docs/setup/production-environment/container-runtimes.md:453]: I agree. I'd prefer to close this pull request.

Comment (RA489) [reply content/en/docs/setup/production-environment/container-runtimes.md:454]: @bart0sh I have updated. PTAL.

Comment (bart0sh) [reply content/en/docs/setup/production-environment/container-runtimes.md:454]: @RA489 thanks for the update. I like it better than current version as it shows which section of the configuration file to update.

**Review by bart0sh — APPROVED** [review]

**Pull request merged**